tests/unit/pyrflx_test.py

```diff
@@ -1203,3 +1204,53 @@ def test_always_valid_aspect(  # pylint: disable=invalid-name
     assert message_always_valid_aspect_value.get("F1") == "RFLX_UNKNOWN_ENUM_1"
     assert message_always_valid_aspect_value.get("F2") == "RFLX_UNKNOWN_ENUM_2"
     assert message_always_valid_aspect_value.get("F3") == 171
+
+
+def test_get_all_sdu_messages(
+    array_message_with_sdu_value: MessageValue, in_array_message_package: Package
+) -> None:
+    array_message_one = in_array_message_package["Foo"]
+    array_message_one.set("Byte", 5)
+    array_message_two = in_array_message_package["Foo"]
+    array_message_two.set("Byte", 6)
+    sdu_message = in_array_message_package["Foo"]
+    sdu_message.set("Byte", 7)
+
+    array_contents: Sequence[TypeValue] = [array_message_one, array_message_two]
+    array_message_with_sdu_value.set("Length", 2)
+    array_message_with_sdu_value.set("Bar", array_contents)
+    array_message_with_sdu_value.set("Payload", sdu_message.bytestring)
+
+    assert array_message_with_sdu_value.valid_message
+
+    all_contained_messages = array_message_with_sdu_value.inner_messages()
+    assert len(all_contained_messages) == 3
+    assert all(isinstance(m, MessageValue) for m in all_contained_messages)
+    assert {m.get("Byte") for m in all_contained_messages} == {5, 6, 7}
```

Comment:
The naming should be consistent. In `typevalue` it is called `inner_message`, here is `sdu_message` as well as `contained_message` used.

Reply:
Fixed in 9f64f90